orttraining/orttraining/python/training/ortmodule/_training_manager.py

```diff
@@ -190,8 +196,11 @@ def backward(ctx, *grad_outputs):
 
                 # Fast version: all backward_outputs are converted first.
                 # This version only works if backward_outputs is an OrtValueVector.
-                transfered_backward_outputs = _utils._ortvalues_to_torch_tensor(backward_outputs, self._device)
-                return tuple(transfered_backward_outputs[idx] if idx != -1 else None for idx in self._gradient_map)
+                transferred_backward_outputs = _utils._ortvalues_to_torch_tensor(backward_outputs, self._device)
+
+                self._rt_inspector.inspect_memory(Phase.POST_BACKWARD)
+
+                return tuple(transferred_backward_outputs[idx] if idx != -1 else None for idx in self._gradient_map)
```

Comment:
typo here

```suggestion
                transfered_backward_outputs = _utils._ortvalues_to_torch_tensor(backward_outputs, self._device)

                self._rt_inspector.inspect_memory(Phase.POST_BACKWARD)

                return tuple(transfered_backward_outputs [idx] if idx != -1 else None for idx in self._gradient_map)
```

Reply:
The repeated "r" should be correct.

Reply:
Ah, sorry, guys. My mistake. :)

Reply:
OK, not a mistake. transferred_backward_outputs  is correct one. I will revert it back.